generators/bdk/python/templates/.gitignore.tpl

```diff
@@ -0,0 +1,125 @@
+# Editors
```

Comment:
that's a lot of things? maybe we can keep it smaller just ignoring what is generated by the project and really common like IDE files, DS_store,...

Reply:
but at the same time it is scary because I see the while diff here, if you never open this file it is probably fine...

Reply:
okay I will simplify this file